components/calendar/README.md

```diff
@@ -23,17 +23,15 @@ Note: All `*-value` properties should be in ISO 8601 calendar date format (`YYYY
 | `max-value` | String | Maximum valid date that could be selected by a user. |
 | `min-value` | String |  Minimum valid date that could be selected by a user. |
 | `selected-value` | String | Currently selected date. |
-| `summary` | String | Summary of the calendar for accessibility. |
+| `summary` | String | ACCESSIBILITY: Summary of the calendar used by screen reader users for identifying the calendar and/or summarizing its purpose. |
 
 ### Events
 
 * `d2l-calendar-selected`: dispatched when a date is selected through click, space, or enter. `e.detail.date` is in ISO 8601 calendar date format (`YYYY-MM-DD`).
 <!-- docs: end hidden content -->
 
-### Accessibility Properties
+## Accessibility
```

Comment:
With calendar there was work done that deviated from the w3c recommendation, which is a bit hard to word in the more matter-of-fact bullet point style we've been using for these sections. Let me know suggestions on if we want that to be more consistent.

Reply:
Should we capitalize `w3c`?

Reply:
Yeah, this does deviate from the matter-of-fact style we use (and not just in bullets) in that it seems to try to explain how the project went, and even calls it the "date picker initiative" which some readers won't have context for (esp. if/when we take the site public) and even a lesson-learned "The calendar implementation emphasized the importance of thorough screen reader testing across all supported options."

I think we need to frame it more simply, without the project context.